src/compiler/test-file/formats/raw/prepare-command-object.js

```diff
@@ -0,0 +1,23 @@
+// NOTE: TestCafe Studio adds additional fields to the command object in RAW tests.
+// They do not affect the execution of the command. Therefore, we should remove them before validation.
+// We should change this mechanism in TestCafe Studio in the future to not add these properties to RAW tests.
+
+const studioRemoveStrategy = () => true;
```

Comment:
We don't expect new properties to remove, so it looks like over-engineering.
It better to replace with more plain code:
```js
enum STUDIO_PROPERTY_NAMES  {
    studio = 'studio',
    selector = 'selector',
};

export default function (commanObj) {
     delete commandObj[STUDIO_PROPERTY_NAMES.studio];

    const selectorValue = commandObj[STUDIO_PROPERTY_NAMES.selector];

    if (isNullOrUnderfined(selectorValue))
        return;

    delete commandObj['selector'];
}
```